src/main.cpp

```diff
@@ -6733,19 +6096,6 @@ bool SendMessages(CNode *pto)
         int64_t nNow = GetTimeMicros();
         requester.DisconnectOnDownloadTimeout(pto, consensusParams, nNow);
 
-        TRY_LOCK(cs_main, lockMain); // Acquire cs_main for IsInitialBlockDownload() and CNodeState()
-        if (!lockMain)
-        {
-            // LOG(NET, "skipping SendMessages to %s, cs_main is locked\n", pto->addr.ToString());
-            return true;
-        }
-        TRY_LOCK(pto->cs_vSend, lockSend);
-        if (!lockSend)
-        {
```

Comment:
excellent..i've wanted to take these out for a while now and came to the same conclusion that they were a contributer to the burstiness of network traffic.